src/js/pre-need/config/form.js

```diff
@@ -488,7 +488,7 @@ const formConfig = {
                     'ui:title': 'Name of deceased'
                   }),
                   'view:cemeteryNumber': {
-                    'ui:title': 'VA National Cemetery where they\'re buried'
+                    'ui:title': 'VA national cemetery where they\'re buried'
```

Comment:
Three (3) instances updated on this page:
![localhost-3001-burials-and-memorials-burial-planning-application-burial-benefits 2](https://user-images.githubusercontent.com/16051172/32812753-867a3700-c95b-11e7-9850-cb9c56150e1c.png)